wasm_samples/mint_rose_trigger/src/lib.rs

```diff
@@ -15,16 +15,15 @@ getrandom::register_custom_getrandom!(iroha_trigger::stub_getrandom);
 
 /// Mint 1 rose for owner
 #[iroha_trigger::main]
-fn main(id: TriggerId, owner: AccountId, _event: EventBox) {
-    let rose_definition_id = "rose#wonderland".parse().unwrap();
-    let rose_id = AssetId::new(rose_definition_id, owner);
+fn main(host: Iroha, context: Context) {
```

Comment:
```suggestion
fn main(host: Iroha) {
```

since `host` is a handle to interact with the host environment it might make more sense to put `context` inside it instead. What do you think?

Reply:
in this case extracting fields of context might be problematic

Reply:
IMO I think it's good as it is